### PR TITLE
Adds script to delete release apks after 72 hours 

### LINKED ISSUE #524: Delete apks from server after 72 hours
We are using a lot of space by storing all the generated apk's and json files on the server, so we need to get rid of them. As @mariobehling suggested we need to delete them after 72 days of the date of generation.
Add the time period in the ```config.json``` created.

### PR DESCRIPTION
**Pull request by mananwason**
Fixes #524. @the-dagger Please review and merge

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Build Server Requirements
...
 * **[upload.php](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/upload.php)** A php script that will upload the json zip that is chosen by the user into a unique folder based on his timestamp.<br>  **[api.php](https://github.com/fossasia/open-event-android/blob/development/apk-generator/api/api.php)** A php script that serves as an API and handles incoming requests to the server.
 * **[copyapk.sh](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/copyApk.sh)** A bash script that will copy the generated and signed apk to the release folder in `public-html` of the server.
+* **[cleanup.sh](https://github.com/fossasia/open-event-android/blob/development/apk-generator/scripts/cleanup.sh)** A bash script that will delete all the signed apks from the release folder in `public-html` of the server. This is to be run as a cronjob using the command ``` crontab 01 00 * * * root Path_to_cleanup.sh ``` where 01 00 means it'll run at 1.00 AM every night and will execute cleanup.sh.
```
Comment (the-dagger) [inline docs/INSTALLATION.md]: What if user generates the app at 12:59?

Comment (mananwason) [inline docs/INSTALLATION.md]: @the-dagger This job runs at 1 AM and will iterate over all folders and delete the ones that were created 3 days back

Comment (the-dagger) [inline docs/INSTALLATION.md]: Ohh, LGBT then.
Merging :+1:

**Comment by the-dagger** [comment]
@mananwason could you please resolve conflicts? 
Because my pr made changes to the documentation. 
Also please add the instruction about running the cron task to all the docs and not just `Installation.md`

File: apk-generator/scripts/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+find /var/www/html/release* -mtime +3 -exec rm -r {} \;
```
Comment (the-dagger) [inline apk-generator/scripts/cleanup.sh]: Also go through the respective folder in /var/www/files/* and /var/www/html/uploads?

**Comment by the-dagger** [comment]
@mananwason please see the comments.

File: apk-generator/scripts/cleanup.sh (hunk 2 above)
Comment (mananwason) [inline apk-generator/scripts/cleanup.sh]: @the-dagger We can delete those after the apk is generated.

**Comment by mananwason** [comment]
@the-dagger Resolved conflicts

**Pull request merged**